src/pkgcheck/objects.py

```diff
@@ -171,8 +171,7 @@ def info(self):
     def filter(self):
         """Mapping of default result filters."""
         from . import results
-        return ImmutableDict({
```

Comment:
What are the effects of this removal of `latest` filter? Does it affect regular usage?
(Note: I haven't understood till now how `latest` filter works in pkgcheck, so asking just in case)

Reply:
OK, thanks for clarification